android/app/src/main/AndroidManifest.xml

```diff
@@ -3,11 +3,8 @@
   <uses-permission android:name="android.permission.CAMERA" />
   <uses-permission android:name="android.permission.USE_BIOMETRIC" />
   <uses-permission android:name="android.permission.USE_FINGERPRINT" />
-  <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
-  <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
-  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
```

Comment:
leaving only the write permission here so hopefully Google Play will approve us now